package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "2.0.0",
```

Comment:
Thank you for review!
I was not aware of it, so I was a little bit surprised that version in my package.json had been updated to 2.0.1... I know the reason why now.
I have changed destination of merge to 1.10.x. But I guess it will not be enough, as I will need a 1.10.x package.json file to be updated to 1.10.y... Maybe I will need to decline the PR and create a new one? We will see

Reply:
Yes, I think it's better to get branch from 1.10.x and transfer the new code there, as the versions and dependencies could mess up with the previous version. Sorry!